Meta/CMake/vcpkg/overlay-ports/angle/cmake-buildsystem/PlatformMac.cmake

```diff
@@ -30,6 +32,7 @@ if(USE_METAL)
 
     list(APPEND ANGLEGLESv2_LIBRARIES
         "-framework Metal"
+        SPIRV-Headers::SPIRV-Headers
```

Comment:
Do we still need the SPIRV-Headers package after you've bumped this up 2k revisions?